Makefile

```
@@ -145,6 +147,18 @@ build:  ## build the edxapp production image
 	$(COMPOSE) build nginx
 .PHONY: build
 
+check-root-user:  ## Make sure the user calling this is not currently root
+	@if [ $(shell id -u) -eq 0 ]; \
```

It is generally safer to use the double bracket syntax:

```suggestion
	@if [[ $(shell id -u) -eq 0 ]]; \
```

Reference: http://mywiki.wooledge.org/BashGuide/Practices#Bash_Tests